src/xo-app/tasks/index.js

```diff
@@ -30,9 +37,26 @@ export const TaskItem = connectStore(() => ({
 }))(({ task, host }) => <Row>
   <Col mediumSize={6}>
     {task.name_label} (on <Link to={`/hosts/${host.id}`}>{host.name_label}</Link>)
+    {' ' + Math.round(task.progress * 100)}%
   </Col>
-  <Col mediumSize={6}>
-    <progress className='progress' value={task.progress * 100} max='100'></progress>
+  <Col mediumSize={4}>
+    <progress style={{marginTop: '0.4rem'}} className='progress' value={task.progress * 100} max='100'></progress>
```

Comment:
No, it's not related to SingleLineRow at all. The `progress` element is not vertically centered anyway. This is the only place where we need to do that.